scripts/bfb-install

```diff
@@ -778,7 +781,7 @@ fi
 # Check NIC mode and unbind mlx5_core driver in NIC mode.
 check_nic_mode
 if [ $is_nic_mode -eq 1 -a -n "$pcie_bdf" ]; then
-  echo "$pcie_bdf" > /sys/bus/pci/drivers/mlx5_core/unbind
+  echo "$pcie_bdf" > /sys/bus/pci/drivers/mlx5_core/unbind 2>/dev/null
```

Comment:
Just want to make sure this line to ignore errors is intended .

Reply:
yes, it's intended and was requested from Moshe.